awc/src/builder.rs

```diff
@@ -140,7 +169,27 @@ impl ClientBuilder {
 
     /// Finish build process and create `Client` instance.
     pub fn finish(self) -> Client {
-        Client(Rc::new(self.config))
+        let connector = if self.connector.is_some() {
+            self.connector.unwrap()
```

Comment:
Why isn't it `if let`?

Reply:
Indeed, fixed that with https://github.com/actix/actix-web/pull/1394/commits/3a8b4d70221b7089b356777e73bf77ffb5617492, thank you!